GoMain/src/main/main.go

```diff
@@ -182,11 +187,8 @@ func orchestrationInit() error {
 	restEdgeRouter.Add(ehandle)
 
 	restEdgeRouter.Start()
-
-	if _, err := os.Stat(dataStorageFilePath); err == nil {
-		sd := storagedriver.StorageDriver{}
-		go startup.Bootstrap(dataStorageService, device.Version, &sd)
-	}
+	storageIns := internalapi.GetStorageInstance()
+	go startup.Bootstrap(dataStorageService, device.Version, &storageIns)
```

Comment:
I think we should initiate the storagemgr at the [orchestrationapi](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/src/orchestrationapi/orchestration.go).
Please refer to the [Start](https://github.com/lf-edge/edge-home-orchestration-go/blob/a2359b166b40ed829e8bb771ccfd7af7032d1244/src/orchestrationapi/orchestration.go#L188) function for discoverymgr.